packages/hasura-auth-js/src/types/options.ts

```diff
@@ -48,7 +48,9 @@ export interface DeanonymizeOptions extends RegistrationOptions {
   password?: string
 }
 
-export interface CommonProviderOptions extends RegistrationOptions, RedirectOption {}
+export interface CommonProviderOptions extends RegistrationOptions, RedirectOption {
+  connect?: boolean
```

Comment:
should this be added to CommonProviderOptions? It is only used in the connect endpoint